### PR TITLE
Patch lldb-3.7.1 to build on mac

### PR DESCRIPTION
**Pull request by ihnorton**
ref https://github.com/llvm-mirror/lldb/commit/6fc7df5c414
@Keno

### CODE REVIEW

**Comment by tkelman** [comment]
should this be deletion or addition?

**Comment by ihnorton** [comment]
Must have left out `-R` the last time I ran diff. Updated, thanks.

File: deps/lldb-3.7.1.patch [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+ endif
+ 
++ifeq ($(HOST_OS),NetBSD)
++$(eval $(call DIR_SOURCES,posix))
++$(eval $(call DIR_SOURCES,netbsd))
++endif
++
+ ifeq ($(HOST_OS),MingW)
+ $(eval $(call DIR_SOURCES,windows))
++SOURCES += posix/ConnectionFileDescriptorPosix.cpp
```
Comment (tkelman) [inline deps/lldb-3.7.1.patch]: the posix here concerns me, mingw is not posix

Comment (ihnorton) [inline deps/lldb-3.7.1.patch]: That is from upstream, but I think it was committed directly: https://www.mail-archive.com/lldb-commits@lists.llvm.org/msg03413.html

I'll take it out, although I don't have a windows build right now to test. ... I'm not sure if anyone else actually builds LLVM with MinGW and Makefiles any more.

Comment (tkelman) [inline deps/lldb-3.7.1.patch]: I've never built lldb so no idea whether the file name concern is justified.

**Pull request merged**